src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -29,6 +29,17 @@ using node::BlockManager;
 using node::KernelNotifications;
 using node::SnapshotMetadata;
 
+static std::vector<Chainstate*> GetAll(const ChainstateManager& chainman)
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/30214#discussion_r1677599944

> Why do you add this function around here instead of using `m_chainstates` as well?

Dropped this function now, it was not hard to get rid of.